ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -558,6 +548,16 @@ else
 				fi
 			fi
 		fi
+
```

Comment:
@bkueng , I just pushed up a commit to this PR that places this logic block a few lines lower.  I did this to reveal what looks like a logic error when compared with a few lines above, `if [ $OUTPUT_MODE != fmu ]`.

Because `if [ $OUTPUT_MODE != fmu ]` is contained in the else statement, I think it could be taken out of the if statement that contains is, then we would have `if [ $OUTPUT_MODE != fmu ]` followed by `if [ $OUTPUT_MODE == fmu ]` that do exactly the same thing.

Let me know your thoughts on this!